src/website/design/content-redesign-notes.md

```diff
@@ -0,0 +1,92 @@
+# Website Layout
+
+This document provides an overview of the content design for the revised openapis.org website.
+
+## Menu
+
+This section provides the content of the menu bar.
+
+### Drop downs
+
+The following are the drop downs we should aim for.
+
+```text
+Learn
+├── What Is OpenAPI? (Relocate existing page).
```

Comment:
```suggestion
├── What Is OpenAPI Initiative? (Relocate existing page).
```
To avoid confusion with the specification itself. We can explain all 3 specs briefly on that page.